src/h3lib/lib/linkedGeo.c

```diff
@@ -103,35 +106,27 @@ LinkedGeoCoord* addLinkedCoord(LinkedGeoLoop* loop, const GeoCoord* vertex) {
  * @param polygon Pointer to the first polygon in the structure
  */
 void H3_EXPORT(destroyLinkedPolygon)(LinkedGeoPolygon* polygon) {
-    LinkedGeoPolygon* currentPolygon = polygon;
-    LinkedGeoPolygon* nextPolygon;
-    LinkedGeoLoop* currentLoop;
-    LinkedGeoLoop* nextLoop;
-    LinkedGeoCoord* currentCoord;
-    LinkedGeoCoord* nextCoord;
     // flag to skip the input polygon
-    int skip = 1;
-    while (currentPolygon != NULL) {
-        currentLoop = currentPolygon->first;
-        while (currentLoop != NULL) {
-            currentCoord = currentLoop->first;
-            while (currentCoord != NULL) {
+    bool skip = true;
+    for (LinkedGeoPolygon *currentPolygon = polygon, *nextPolygon;
```

Comment:
Nit: I believe our style in this project is `LinkedGeoPolygon* currentPolygon`, can we use that here? Or would that require `LinkedGeoPolygon* currentPolygon, LinkedGeoPolygon* nextPolygon`?

Reply:
You mean the pointer alignment? Ya it's a problem. `clang-format` isn't even fixing it. The only alternative is `LinkedGeoPolygon* nextPolygon;` followed by `for (LinkedGeoPolygon* currentPolygon = ...)`. I once answered a question about this on StackOverflow here: https://stackoverflow.com/a/16490876/1930331.